app/components/EntryImages.tsx

```diff
@@ -0,0 +1,30 @@
+import { type LoadedDataType } from "~/routes/entries/$headword"
+
+interface EntryProps {
+  data: LoadedDataType
+}
+
+const Entry = ({ data }: EntryProps): JSX.Element => {
+  // TODO: replace array contents with legitimate links
+  const testUrlArray = [...Array(3)].map((elm) => {
+    return "https://picsum.photos/300/200"
+  })
+
```

Comment:
So we have yet to implement this. I'm leaving this here for visual purposes. I'm thinking rather than changing the shape of the current `LoadedDataType` (which is what a `getEntries` function returns), I'm going to create another method to get entry images by headword.

Reply:
Or, we could just join them from the `det_images` table, which we don't seem to have in `schema.prisma` — that might be easier? But this is fine for now.